nservicebus/ravendb/manual-dtc-settings.md

```diff
@@ -0,0 +1,62 @@
+
+## Definitions
+
+Support for the Distributed Transaction Coordinator (DTC) in RavenDB is dependent upon the **ResourceManagerId** and **TransactionRecoveryStorage** settings.
+
+### ResourceManagerId
+
+The ResourceMangaerId is a Guid that uniquely identifies a transactional resource on a machine, and must be both unique on that machine, and stable (deterministic) between restarts of the process utilizing the DTC. If more than one RavenDB document store attempts to use the same ResourceManagerId, it can result in the following error during a commit operation:
+
+> "A resource manager with the same identifier is already registered with the specified transaction coordinator"
+
+### TransactionRecoveryStorage
+
+To guard against the loss of a committed transaction, RavenDB requires a storage location to persist data in the event of a process crash immediately following a transaction commit.
+
+RavenDB provides the following methods for persisting transaction recovery storage information:
+
+* `VolatileTransactionRecoveryStorage` persists the information in memory, and since the information will be lost during a process restart, is not a safe storage method.
+* `IsolatedStorageTransactionRecoveryStorage` persists the information in [Isolated Storage](https://msdn.microsoft.com/en-us/library/system.io.isolatedstorage.aspx). This is the default for NServiceBus.RavenDB when connecting with a connection string, as it does not require any additional configuration. However, in high-contention scenarios, IsolatedStorage has been found to become unstable and may begin to throw exceptions. Therefore using LocalDirectoryTransactionRecoveryStorage is preferable.
+* `LocalDirectoryTransactionRecoveryStorage` stores transaction recovery information to a local path on disk, and is the safest option for production scenarios, but must be configured with a directory that is writeable by the application.
+
+## Configuring safe settings
+
+In order to configure safe settings for production use of RavenDB, construct a `DocumentStore` instance and configure the `ResourceManagerId` and `TransactionRecoveryStorage` properties as shown in the following code:
+
+snippet:RavenDBManualDtcSettingExample
+
+In order to provide transaction safety, the following must be observed:
+
+* `ResourceManagerId` must be constant across process restarts, and uniquely identify the process running on the machine. (Do not use `Guid.NewGuid()`.) Otherwise, the transaction recovery process will fail when the process restarts.
+* `TransactionRecoveryStorage` must be set to an instance of `LocalDirectoryTransactionRecoveryStorage`, configured to a directory that is constant across process restarts, and writeable by the process.
```

Comment:
above u say application. here you say process. also why is that text duplicated?